PWGJE/EMCALJetTasks/AliAnalysisTaskPWGJEQA.cxx

```diff
@@ -153,16 +161,47 @@ void AliAnalysisTaskPWGJEQA::UserCreateOutputObjects()
 {
   // Create histograms
   AliAnalysisTaskEmcalJet::UserCreateOutputObjects();
+
+  // Intialize AliEventCuts
+  if (fUseAliEventCuts) {
+    fEventCutList = new TList();
+    fEventCutList ->SetOwner();
+    fEventCutList ->SetName("EventCutOutput");
+
+    fEventCuts.OverrideAutomaticTriggerSelection(fOffTrigger);
+    if(fUseManualEventCuts==1)
+    {
+    		fEventCuts.SetManualMode();
+    		fEventCuts.fMC = MCEvent(); //before was= false
+    	if(fForceBeamType != kpp)
```

Comment:
spacing looks funny here, and in a few other places...try to fix it if you easily can